requirements.txt

```diff
@@ -44,6 +44,10 @@ git+https://github.com/CenterForOpenScience/modular-odm.git@develop
 # Python markdown extensions for comment emails
 git+git://github.com/aleray/mdx_del_ins.git
 
+# Kombu with the ability to specify queue priority
+# TODO: Remove this when Kombu has a stable release including commit c20f854
+git+git://github.com/mfraezz/kombu.git@v3.0.36
```

Comment:
We're trying to move away from using forks on devs' accounts. When you get a chance, can you transfer ownership of your fork to me so I can then transfer it to the COS org account?